### DISCUSSION TITLE
Does AtlasMap supports automatic mapping if key is same on both source & target

### DISCUSSION THREAD

**Discussion by gurumurthy-godlaveeti** [atlasmap/atlasmap#4140] [Q&A]
Hi,

I would like to see if any possibility to have default mapping for all fields having same name & structure on both source & target sides.

For example, messageHeader is same in both source & target as per below screenshot so can it have automatic mapping?
![image](https://user-images.githubusercontent.com/101185013/176636613-f57b4c7e-bf73-45f4-9437-3b2980bd9a58.png)

**Comment by igarashitm** [comment]
There's an open issue for that - https://github.com/atlasmap/atlasmap/issues/86
Contribution is welcome.
